router-tests/.golangci.yml

```diff
@@ -0,0 +1,27 @@
+version: "2"
+linters:
+  enable:
+    - errcheck
+    - paralleltest
+    - govet
+    - staticcheck
+    - ineffassign
+    - unused
+  settings:
+    errcheck:
+      exclude-functions:
+        - (io.ReadCloser).Close
+        - (*net/http.Server).Close
+        - (net/http.ResponseWriter).Write
+        - (*github.com/gorilla/websocket.Conn).Close
+        - (*compress/gzip.Reader).Close
+        - (io.Closer).Close
+        - (*mime/multipart.Writer).Close
+    paralleltest:
+      ignore-missing: false
+      ignore-missing-subtests: false
```

Comment:
I do not recommend this linter at the moment, it just makes a lot of yellow and there is precedent discussion for not wanting parallelization for the sake of parallelization.

Reply:
I agree with @endigma here. Adding parallelization should be reasoned. [In the worst case](https://threedots.tech/post/go-test-parallelism/), it could even make them slower. We have already run into a resource exhaustion scenario.

Regarding MCP @alepane21 I did it on purpose to not further stress out port allocation. Making them parallel didn't speed them up much.